vue/src/components/LayerSelection.vue

```diff
@@ -11,6 +11,56 @@ const route = useRoute();
 const proposals = computed(() => route.path.includes('proposals'));
 
 
+const modelRunEnabled = computed(() => state.openedModelRuns.size > 0);
+
+// Either has ground truth, is ground truth, or no ground truth
+const groundTruthState = computed<'NoGroundTruth'|'HasGroundTruth'|'AllGroundTruth'>(() => {
+  let result: 'NoGroundTruth'|'HasGroundTruth'|'AllGroundTruth'  = 'NoGroundTruth';
+  if (scoringApp.value) {
+    return 'HasGroundTruth';
+  }
+  if (proposals.value) {
+    if (state.proposals.ground_truths) {
+      return 'HasGroundTruth';
+    } else {
+      return 'NoGroundTruth';
+    }
```

Comment:
Would be cleaner to use an enum here, imo (not that this is as big a deal as if it were python, as the TS compiler would catch any string literal typos anyway, but I think it's slightly cleaner).

```suggestion
enum GroundTruthState {
  NoGroundTruth = 0,
  HasGroundTruth = 1,
  AllGroundTruth = 2,
}

// Either has ground truth, is ground truth, or no ground truth
const groundTruthState = computed<GroundTruthState>(() => {
  let result: GroundTruthState  = GroundTruthState.NoGroundTruth;
  if (scoringApp.value) {
    return GroundTruthState.HasGroundTruth;
  }
  if (proposals.value) {
    if (state.proposals.ground_truths) {
      return GroundTruthState.HasGroundTruth;
    } else {
      return GroundTruthState.NoGroundTruth;
    }
```

(Would also need to update rest of the file)